pyproject.toml

```diff
@@ -16,15 +16,24 @@ whitenoise = {extras = ["brotli"], version = "^6.6.0"}
 gunicorn = "^22.0.0"
 uvicorn = "^0.29.0"
 django-bootstrap5 = "^24.2"
+freezegun = "^1.5.1"
+django-hint = "^0.3.2"
```

Comment:
are they the main project dependencies? if not, could you please move them to the `dev` group?